packages/wallet/src/test-utils/launchNode.test.ts

```diff
@@ -14,6 +16,22 @@ test('launchNodeAndGetWallets - empty config', async () => {
   stop();
 });
 
+test('launchNodeAndGetWallets - custom config', async () => {
+  // #region launchNode-custom-config
+  const { stop, provider } = await launchNodeAndGetWallets({
+    launchNodeOptions: {
+      chainConfigPath: path.join(cwd(), '.fuel-core/configs/chainConfig.json'),
+      args: ['--manual_blocks_enabled'],
+    },
+  });
+  const {
+    consensusParameters: { gasPerByte },
+  } = provider.getChain();
+  expect(gasPerByte.toNumber()).toEqual(4);
+  stop();
```

Comment:
Just trying to improve code readability by adding some empty space around it.

Feel free to ignore it.

```suggestion
  const chainConfigPath = path.join(cwd(), '.fuel-core/configs/chainConfig.json');
  
  const { stop, provider } = await launchNodeAndGetWallets({
    launchNodeOptions: {
      chainConfigPath,
      args: ['--manual_blocks_enabled'],
    },
  });
  
  const { consensusParameters: { gasPerByte } } = provider.getChain();
  
  expect(gasPerByte.toNumber()).toEqual(4);
  
  stop();
```